discovery-infra/test_infra/tools/assets.py

```diff
@@ -47,3 +65,83 @@ class NetworkAssets(Assets):
 
     def __init__(self):
         super().__init__(assets_file=consts.TF_NETWORK_POOL_PATH)
+
+
+class DynamicNetworkAssets(NetworkAssets):
+    """ An assets class that stores values based on the current available
+        resources, in order to allow multiple (but not parallel) installations
+        while avoiding conflicts. """
+
+    IP_NETWORK_FIELDS = (
```

Comment:
We might need this in consts so that we can use the strings elsewhere in the project.